scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -283,7 +283,7 @@
         _ParamSpec('string', 'parameter_server',
```

Comment:
can you update the comment before this option to include a horovod: section, including a link to the github repo and any other info that is relevant?

Reply:
for example, maybe a desciption of what a setup with m workers and n gpus per worker looks like in the horovod case (how many copies of the benchmark run, for example).